src/scripts/modal.js

```diff
@@ -0,0 +1,29 @@
+// Get the modal
+var modal =
+	document.getElementById("myModal");
+
+// Get the button that opens the modal
+var btn = document.getElementById("myBtn");
+
+// Get the <span> element that closes the modal
+var span =
+	document.getElementsByClassName(
+		"close"
+	)[0];
+
```

Comment:
you can instead build a custom hook like useCloseModal and use "useref" instead of doing vanilla js. this it will be much more useable.